src/js/controller/PaletteController.js

```diff
@@ -106,11 +110,21 @@
     } else {
       colorPicker.spectrum('set', color);
     }
-    this.setTitleOnPicker_(color, colorPicker);
   };
 
-  ns.PaletteController.prototype.setTitleOnPicker_ = function (title, colorPicker) {
-    var spectrumInputSelector = '.sp-replacer';
-    colorPicker.next(spectrumInputSelector).attr('title', title);
+  /**
+   * @private
+   */
+  ns.PaletteController.prototype.showValueDisplayer_ = function (evt) {
```

Comment:
![multi-line tooltip](https://cloud.githubusercontent.com/assets/2543511/13468320/5cf1e5fc-e0a2-11e5-9c48-9b158ed0b16c.png)
I can't say I like it.